lib/client.js

```diff
@@ -7,6 +7,9 @@ var PROMISE_TIMEOUT = 5000, // 5 seconds
     pendingPromises = {},
     ids             = {};
 
+// export Promise!
+window.Promise = Promise;
```

Comment:
Could we namespace it under ZAFClient perhaps, to avoid making it seem like the browser supports it?

Reply:
Isn't that the point of a polyfill? Making it so that it's like the browser supports it?

Reply:
It should be that the polyfill is present to provide a consistent API in v2 apps, but that the polyfill should no-op if the browser already supports Promise.  The polyfill we use is spec compliant, so it should be transparent

Reply:
It just seems a bit intrusive to **implicitly** polyfill a native API, but I admit I can't think of any actual problems this could cause, so happy to 👍

Reply:
It's possible developers are already overwriting this, so we shouldn't rely on it within our code.

Reply:
Should it be `window.Promise = window.Promise || Promise;`

Reply:
@svizzari yes, but in the code we already to that:

```
Promise = window.Promise || require('../vendor/native-promise-only'),
(...)
window.Promise = Promise;
```